sklearn/multioutput.py

```diff
@@ -45,18 +45,18 @@ def __init__(self, estimator, n_jobs=1):
 
     def fit(self, X, y, sample_weight=None):
         """ Fit the model to data.
-        Fits a seperate model for each output variable.
+        Fit a seperate model for each output variable.
 
         Parameters
         ----------
-        X : (sparse) array-like, shape = [n_samples, n_features]
+        X : (sparse) array-like, shape = (n_samples, n_features)
```

Comment:
Actually, I am sorry for requiring so many passes. Thanks for patiently reviewing this.